crypto/signatures_test.go

```diff
@@ -65,11 +180,11 @@ func TestSigning(t *testing.T) {
 	// worked around 98% of the time. Tests would usually pass, but 200
 	// iterations would normally cause a failure.
 	for i := 0; i < iterations; i++ {
-		// Generate the keys.
-		sk, pk, err := GenerateSignatureKeys()
-		if err != nil {
-			t.Fatal(err)
-		}
+		// Create dummy key pair.
+		var entropy [EntropySize]byte
+		entropy[0] = 0x05
+		entropy[1] = 0x08
```

Comment:
I'm just used to assigning hex values to byte types, but I can do normal base-10 if you guys prefer. Fixed.